action.yml

```diff
@@ -70,6 +74,8 @@ runs:
         INPUT_TOOL_NAME: ${{ inputs.tool_name }}
         INPUT_USE_BUNDLER: ${{ inputs.use_bundler }}
         INPUT_WORKDIR: ${{ inputs.workdir }}
+        BASE_REF: ${{ github.event.pull_request.base.sha }}
+        HEAD_REF: ${{ github.sha }}
```

Comment:
First off all thanks for this PR, it seems like a really useful addition that could speed up our github action by a lot as using rubocop takes about 5 minutes currently to run over the whole repo.

Should this look at `github.event.pull_request.base.sha` instead?

I tried out your branch in a github action workflow but the HEAD_REF commit seemed wrong and gave me the following error: `fatal: Invalid revision range`

Reply:
I guess you need git fetch.

https://github.com/reviewdog/action-rubocop/pull/103/files#diff-b803fcb7f17ed9235f1e5cb1fcd2f5d3b2838429d4368ae4c57ce4436577f03fR14

Reply:
Do you mean using `github.event.pull_request.head.sha` for `HEAD_REF`?

The idea of this change is to give to rubocop only files introduced in the PR branch, but we don't want to also give to rubocop files that changed on the base branch especially if it diverged a lot.

This could be resolved in different ways, but in current implementation the idea is to also fetch bare minimum to not slowdown by fetching, so usage of `actions/checkout` with default `depth: 1` is assumed.

Also `github.sha` is pointing at automatically created merge commit of base into head ([docs](https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#pull_request)), while `github.event.pull_request.head.sha` points at the PR branch head.

If we would have all commits available it would be simpler to do `diff base...head` (note [three dots](https://git-scm.com/docs/git-diff#Documentation/git-diff.txt-emgitdiffemltoptionsgtltcommitgtltcommitgt--ltpathgt82308203-1)) where head can be both merge commit or the tip of the branch, but if we fetch only two tips of the branches, `diff base...head` is not possible, as not all needed commits are available.
Otherwise `diff base..merge-head` should give much smaller diff then `diff base..head`.

So most probable cause of `fatal: Invalid revision range` is because your workflow either doesn't fetch the tip of the base branch or fetches not the automatically created merge head (default using actions/checkout).